packages/components/src/components/page/page.scss

```
@@ -18,15 +18,15 @@
 		display: flex;
 		flex-direction: column;
 
-		header,
-		footer {
+		header:first-of-type,
+		footer:first-of-type {
 			display: flex;
 			flex: 0 1 auto;
 			flex-grow: 0;
 			flex-shrink: 0;
 		}
 
-		main {
+		main:first-of-type {
```

```suggestion
		> .db-main {
```

@nmerget I assume that you mean by this the first `main` within the document or directly underneath the page. But any nested `main` would also be targeted by this selector, as it mainly means "on the same level", compare to e.g. https://jsbin.com/xetuxovoru/edit?html,css,output

---

fixed